<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideWidjet</class>
 <widget class="QWidget" name="SideWidjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>95</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnNew">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>&amp;New</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>&amp;Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnDelete">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>&amp;Delete</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnExit">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0.882157, y1:0.142, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>E&amp;xit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
